linters/djlint/djlint.test.ts

```diff
@@ -0,0 +1,4 @@
+import { linterCheckTest } from "tests";
+
+// TODO(Tyler): We will eventually need to add a couple more test cases involving failure modes and other packages.
+linterCheckTest({ linterName: "djlint" });
```

Comment:
Please rename `logout.html` -> `logout.in.html` in order for the test to pick it up (and then run `npm test djlint` locally to generate the snapshot, which acts as the expected output. For more info, see `tests/readme.md`